lib/Conversion/TritonGPUToLLVM/PtxAsmFormat.cpp

```diff
@@ -74,6 +76,25 @@ SmallVector<PTXBuilder::Operand *, 4> PTXBuilder::getAllArgs() const {
   return res;
 }
 
+mlir::Value PTXBuilder::launch(ConversionPatternRewriter &rewriter,
+                               Location loc, Type resTy, bool hasSideEffect,
+                               bool isAlignStack,
```

Comment:
It looks `isAlignStack` is not consumed anywhere?

Reply:
Yeah, I've corrected the code.